doc/man3/STORE_LOADER.pod

```diff
@@ -0,0 +1,192 @@
+=pod
+
+=head1 NAME
+
+STORE_LOADER, STORE_LOADER_CTX, STORE_LOADER_new,
+STORE_LOADER_set0_scheme, STORE_LOADER_set_open,
+STORE_LOADER_set_ctrl, STORE_LOADER_set_load, STORE_LOADER_set_eof,
+STORE_LOADER_set_close, STORE_LOADER_free, STORE_register_loader,
+STORE_unregister_loader, STORE_open_fn, STORE_ctrl_fn, STORE_load_fn,
+STORE_eof_fn, STORE_close_fn - Types and functions to manipulate,
+register and unregister STORE loaders for different URI schemes 
+
+=head1 SYNOPSIS
+
+ #include <openssl/store.h>
+
+ typedef struct store_loader_st STORE_LOADER;
```

Comment:
Are these typedefs appropriate for the man page? I don't think we do that for other OpenSSL types.